### PR TITLE
astyle: update to 3.4.8

### PR DESCRIPTION
**Pull request by szhorvat**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 13.5.2 22G91 arm64
Xcode 15.0 15A240d

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [x] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by szhorvat** [comment]
I checked that the build completed with `port install astyle +java` (assuming I installed `port install jdk), but I am not very familiar with Java or JNI, so consider this incompletely tested.

File: devel/astyle/Portfile [hunk 1]
```diff
@@ -18,15 +18,13 @@ long_description    Artistic Style is a source code indenter, source code format
                     programming languages.
 
 homepage            http://astyle.sourceforge.net
-master_sites        sourceforge:project/astyle/astyle/astyle%20${version}
-distname            ${name}_${version}_macos
-checksums           rmd160  5aed3c48b59aedf02e247652db6fad597ef9bd63 \
-                    sha256  c4eebbe082eb2cb98f90aafcce3da2daeb774dd092e4cf8b728102fded8d1dcf \
-                    size    187918
+master_sites        sourceforge
```
Comment (ryandesign) [inline devel/astyle/Portfile:21]: Please revert this change to `master_sites`. See explanation at https://trac.macports.org/wiki/howto/AvoidRedirects.

You'll need to change `${version}` to `${branch}` which you'll have to define first: https://trac.macports.org/wiki/PortfileRecipes#branch

Comment (szhorvat) [reply devel/astyle/Portfile]: Thanks for the explanation! This is corrected now.

**Comment by szhorvat** [comment]
Last force-push is to fix the livecheck regex

File: devel/astyle/Portfile [hunk 2]
```diff
@@ -18,15 +19,13 @@ long_description    Artistic Style is a source code indenter, source code format
                     programming languages.
 
 homepage            http://astyle.sourceforge.net
```
Comment (herbygillot) [inline devel/astyle/Portfile:21]: Please also set the homepage URL to HTTPS

Comment (szhorvat) [reply devel/astyle/Portfile]: Done

Comment (herbygillot) [reply devel/astyle/Portfile]: Thanks
